### PR TITLE
doc: releases: updates for octopus

### PR DESCRIPTION
**Pull request by theanalyst**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
`/home/jenkins-build/build/workspace/ceph-pr-docs/doc/releases/general.rst:1: WARNING: Unknown target name: "15.2.3".`

**Comment by theanalyst** [comment]
> `/home/jenkins-build/build/workspace/ceph-pr-docs/doc/releases/general.rst:1: WARNING: Unknown target name: "15.2.3".`

done thanks!

File: doc/releases/octopus.rst [hunk 1]
```diff
@@ -1,3 +1,104 @@
...
+This is the third bug-fix release of the Ceph Octopus stable release series.
+This release mainly is a workaround for a potential OSD corruption in v15.2.2.
+We advise users to upgrade to v15.2.3 directly. For users running v15.2.2 please
+execute the following::
+
+  ceph config set osd bluefs_preextend_wal_files false
+
+Changelog
+---------
+* bluestore: common/options.cc: disable bluefs_preextend_wal_files (`issue#45613 <http://tracker.ceph.com/issues/45613>`_, Neha Ojha)
```
Comment (smithfarm) [inline doc/releases/octopus.rst:13]: Could we change this line to

```
* bluestore: remove preextended WAL support
```

? That's the title of the commit message and it more accurately describes what the fix is doing.

Comment (smithfarm) [reply doc/releases/octopus.rst:13]: see the master PR https://github.com/ceph/ceph/pull/35497

File: doc/releases/octopus.rst [hunk 2]
```diff
@@ -1,3 +1,104 @@
...
+
+This is the third bug-fix release of the Ceph Octopus stable release series.
+This release mainly is a workaround for a potential OSD corruption in v15.2.2.
+We advise users to upgrade to v15.2.3 directly. For users running v15.2.2 please
+execute the following::
+
+  ceph config set osd bluefs_preextend_wal_files false
+
+Changelog
+---------
```
Comment (smithfarm) [inline doc/releases/octopus.rst:12]: can you insert a blank line here?

**Pull request merged**